apps/frontend/src/app/Database/DataManagers/BuildSettingData.ts

```diff
@@ -93,7 +93,7 @@ const initialBuildSettings: BuildSetting = deepFreeze({
   optimizationTarget: undefined,
   mainStatAssumptionLevel: 0,
   allowLocations: [],
-  allowPartial: false,
+  allowPartial: true,
```

Comment:
Sneaking this in here is interesting. Enabling this by default messes up the graph display in an annoying way. Also random since this doesn't look like it should be in this PR

Reply:
This error is actually created by partial builds, so its kind of related. How does this mess up the graph? like crash, or just create clusters at low ranges of the graph?

Reply:
Now realized the annoyance, I guess this should be off by default for now.

Reply:
![image](https://user-images.githubusercontent.com/36019388/221762604-64f2fa13-b4c2-4e92-bb15-3c3e73199eb4.png)
yea just clusters at the bottom end, so y-axis is all messed up